Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .9\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph  \n",
+    "plt.bar(r_values, dist) \n",
+    "plt.xlabel('X', fontsize = 27) \n",
+    "plt.ylabel('P(x)', fontsize = 27)\n",
+    "plt.title('Bionomial Distribution w/ n = 10, p = .9', fontsize = 30)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Seen in the code above with a binomial distrbution, it's almost impossible to succeed less than five times in a 10 trial grouping. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C: Does a greater sample size of 15 have a larger expetected value than a sample size of 10?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 168,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "5.0\n",
+      "8.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .5\n",
+    "print (stats.binom(n, p).ppf(.5))\n",
+    "n = 15\n",
+    "print (stats.binom(n, p).ppf(.5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Yes if p is the same, then if one increases the sample size, the expected value will also grow larger. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "D: Since it's not longer continious does that mean for locating the sigmas and x values associated with the probablities one follows the properties of rounding? "
```

Comment:
Part 2 Problem 1D:
- minor typo: continious --> continuous
- I don't understand what this answer is saying and why it's phrased as a question -- if a distribution is discrete, I think this means only integer values of x events are possible and therefore they must exclusively have one corresponding probability and sigma. i.e., I think we are trying to show that in a given binomial distribution, for some x number of events, it will always have one probability of occurring unless n=trials and/or p=probability of success is changed, during an experiment.